DSA-Campaign-Uplift-Estimation/src/main/webapp/Compare/script-compare.js

```diff
@@ -108,32 +116,112 @@ function drawBarGraph() {
 }
 
 function drawTable(DSACampaignList, keywordCampaign) {
-    var data = new google.visualization.DataTable();
-    data.addColumn('string', 'DSA Campaign');
-    data.addColumn('string', 'Start Date');
-    data.addColumn('string', 'End Date');
-    data.addColumn('number', 'Manual CPC');
-    data.addColumn('number', 'Daily Budget');
-    data.addColumn('string', 'Locations');
-    data.addColumn('string', 'Negative Locations');
-    data.addColumn('string', 'Domain');
-    data.addColumn('string', 'Targets');
-    data.addColumn('string', 'Ad Text');
-    data.addColumn('number', 'Impressions Uplift');
-    data.addColumn('number', 'Clicks Uplift');
-    data.addColumn('number', 'Cost Uplift (USD)');
+    var table = document.getElementById('table');
+
+    // create the table
+    var settingsTable = document.createElement("TABLE"); 
+
+    // create the row of headers
+    var headersRow = document.createElement("TR");
+
+    var header1 = document.createElement("TH");
+    header1.appendChild(document.createTextNode("DSA Campaign"));
+    var header2 = document.createElement("TH");
+    header2.appendChild(document.createTextNode("Start Date"));
+    var header3 = document.createElement("TH");
+    header3.appendChild(document.createTextNode("End Date"));
+    var header4 = document.createElement("TH");
+    header4.appendChild(document.createTextNode("Manual CPC"));
+    var header5 = document.createElement("TH");
+    header5.appendChild(document.createTextNode("Daily Budget"));
+    var header6 = document.createElement("TH");
+    header6.appendChild(document.createTextNode("Locations"));
+    var header7 = document.createElement("TH");
+    header7.appendChild(document.createTextNode("Negative Locations"));
+    var header8 = document.createElement("TH");
+    header8.appendChild(document.createTextNode("Domain"));
+    var header9 = document.createElement("TH");
+    header9.appendChild(document.createTextNode("Targets"));
+    var header10 = document.createElement("TH");
+    header10.appendChild(document.createTextNode("Ad Text"));
+    var header11 = document.createElement("TH");
+    header11.appendChild(document.createTextNode("Impressions Uplift"));
+    var header12 = document.createElement("TH");
+    header12.appendChild(document.createTextNode("Clicks Uplift"));
+    var header13 = document.createElement("TH");
+    header13.appendChild(document.createTextNode("Cost Uplift (USD)"));
+
+    headersRow.appendChild(header1);
+    headersRow.appendChild(header2);
+    headersRow.appendChild(header3);
+    headersRow.appendChild(header4);
+    headersRow.appendChild(header5);
+    headersRow.appendChild(header6);
+    headersRow.appendChild(header7);
+    headersRow.appendChild(header8);
+    headersRow.appendChild(header9);
+    headersRow.appendChild(header10);
+    headersRow.appendChild(header11);
+    headersRow.appendChild(header12);
+    headersRow.appendChild(header13);
```

Comment:
This is a lot of repeated code. Can we have a function that takes in a list of header names?

Reply:
Sorry done, this should have been obvious.